docs/UserGuide.md

```diff
@@ -225,14 +225,6 @@ DLTbook comes with sample data to help you get started. Here are some basic comm
 
 - Check our [Troubleshooting Guide](link-to-troubleshooting) if you run into any issues
 
-<box type="tip" seamless>
```

Comment:
Commands were actually case-sensitive all along - nice work for spotting this error in the previous version of the UG! 👍